src/core/localization/gettext/ar/common.po

```diff
@@ -403,10 +403,6 @@ msgstr "Stddev"
 msgid "Median"
 msgstr "الوسيط"
 
-#: advanced_analytics_data_list##filter_nominator
-msgid "Filter Nominator"
```

Comment:
With normal gettext flows this should get a fuzzy flag or get commented out as "obsolete" (commented with `#~` prefix) so that it can resurface later. This can be achieved if you run msgmerge between the newly converted file and the one in repos and not just plainly replace.

Reply:
https://stackoverflow.com/questions/10435926/how-to-automatically-remove-unused-gettext-strings

Reply:
idk why but can't make this command working in short time. 
I've marked by hands for now. it feels like we need to add such command in our chain.